packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/updateQueryString.ts

```diff
@@ -0,0 +1,52 @@
+const getValueWithoutBrackets = (value: string) => {
+    if (value.startsWith('(') && value.endsWith(')')) {
+        return value.slice(1, -1);
+    }
+    return value;
+};
+
+const isContainingSpaces = (value: string) => {
+    return value.indexOf(' ') !== -1;
+};
+
+const toQueryValue = (value: string) => {
+    return isContainingSpaces(value) ? `"${value}"` : value;
+};
+
+// Three cases to consider when this function is called
+// 1 - queryString is empty
+// 2 - fieldName already exists in query string
+// 3 - fieldName does not exist in query string
+export const updateQueryString = (
+    queryString: string | undefined,
+    fieldName: string,
+    value: string,
+) => {
+    // 1 - Empty query string returning directly fieldName:value
```

Comment:
Wondering if these has to be handled as a separate case. Ideally it should be the same as 3? Unless of course queryString can be really undefined (instead of ""), but in that case the comment is inaccurate.

Reply:
Note code is changed a bit, but still a valid question. Skipping the queryString check would mean i get into the `not a match` case and i will get a space in front of the result.

I think it is a bit more readable this (updated) way